egs/swbd/s5c/local/nnet3/tuning/run_tdnn_e.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+# _e is as _d but it uses random offsets generated using within-speaker covariance
+# to perturb data during training.
+# d is as c, but with one extra layer.
```

Comment:
I added results to local/chain/tuning/run_tdnn_7m.sh
Actually this method helps to reduce overtraining on chain and there is no improvement on Xent model. I will include results for xent too.